src/glossary.rst

```diff
@@ -5919,6 +5929,14 @@ unary operator
 :dp:`fls_p6mk2zrwgwem`
 A :dt:`unary operator` operates on one :t:`operand`.
 
+.. _fls_WuLL4SvSKavZ:
+
+undefined behavior
+^^^^^^^^^^^^^^^^^^
+
+:dp:`fls_WpwmltUMQGZa`
+Situations that result in unbounded errors.
```

Comment:
This would need a `:dt:`. How about

Undefined behavior is a situation that results in an unbounded error.